### PR TITLE
akita bug fix & merge new movie list

### PR DESCRIPTION
**Pull request by pldespaigne**
- [x] refactor the org query to properly get movies inside org obj
- [x] refactor movie list component
- [x] properly solve merge conflict from new list component

### CODE REVIEW

File: libs/movie/src/lib/movie/list/list.component.ts [hunk 1]
```diff
@@ -43,4 +40,7 @@ export class ListComponent implements OnInit, OnDestroy {
     this.service.remove(id);
   }
 
+  public trackOrgs(index: number, org: Organization) {
+    return org ? org.id : undefined;
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/list/list.component.ts]: `return org.id`

File: libs/organization/src/lib/+state/organization.query.ts [hunk 2]
```diff
@@ -1,20 +1,39 @@
...
     super(store);
   }
 
   public getOrgId(name: string) {
-    return this.getAll().find(org => org.name === name).id;
+    const orgs = this.getAll({
+      filterBy: org => org.name === name,
+      limitTo: 0
+    });
+    return orgs[0].id;
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/+state/organization.query.ts]: Why ?

File: libs/movie/src/lib/movie/list/list.component.ts (hunk 1 above)
Comment (pldespaigne) [reply libs/movie/src/lib/movie/list/list.component.ts]: - [x] done

File: libs/organization/src/lib/+state/organization.query.ts (hunk 2 above)
Comment (pldespaigne) [reply libs/organization/src/lib/+state/organization.query.ts]: - [x] done

**Pull request merged**